integration-tests/gatsby-source-wordpress/test-fns/test-utils/urling.js

```diff
@@ -0,0 +1,64 @@
+// forked from https://github.com/mayinbun/urling/blob/master/src/index.js
```

Comment:
I copied this over to make a small change (only returning true once we get a 200 status code) - doesn't feel worth PR'ing it back